source/_integrations/rtorrent.markdown

```diff
@@ -70,3 +81,27 @@ monitored_variables:
     active_torrents:
       description: The number of torrents that are actively ( measurable speed ) leeching, seeding or both.
 {% endconfiguration %}
+
+## Switch
+
+The `rtorrent` switch platform allows you to control your [rTorrent](https://rakshasa.github.io/rtorrent/) client from within Home Assistant. The platform enables you switch all your active torrents in pause, and then unpause them all.
+
+To add this switch to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+switch:
```

Comment:
We no longer allow integrations to add or change a platform YAML configuration.

More information on this can be found in Architecture Decision Record:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>

Please note that this integration connects to a device or service and another Architecture Decision Record applies that disallows the use of YAML configuration in favor of a configuration flow via the UI:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

As these changes often involve a bit of work and some significant shift in the current code, we will close this PR for now.

We (and our community!) would really appreciate it if you could start on the refactoring of this integration in a new PR.

Thanks already! :+1: